setup.cfg

```diff
@@ -28,6 +28,7 @@ install_requires =
     # TODO: add a released version of pystiche
     torch >= 1.6
     torchvision >= 0.7
+    more_itertools >= 8.4
```

Comment:
Why the constraint to `>=8.4`. Is this the first release where `pairwise` is included?